### PR TITLE
momentum-ui styling bundling bug

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,84 +29,68 @@
...
-3. Copy the `@momentum-ui/core/css/momentum-ui.min.css` to your styles/css directory.
-4. In the `<head>`, of your HTML reference the location of your momentum-ui.min.css:
-
-    `<link rel='stylesheet' href='path/to/momentum-ui/css/momentum-ui.min.css'>`
-
-***
-### 2. SCSS
-1. Add `$brand-font-folder` and `$icon-font-path` variables to the top of your app's Sass file.
 
+We need to externally import all the fonts, icons, images and core _CSS_ manually. Import `@webex/components/dist/webexComponents.css` into your main entry JavaScript file.
```
Comment (lalli-flores) [inline README.md]: We should also show the alternative way of importing the CSS with a `link` tag on their HTML file

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import './styles/index.scss';
+import './../momentum-ui.scss';
```
Comment (lalli-flores) [inline src/index.js]: If we're just going to import it like this, doesn't it make sense to move `momentum-ui.scss` into `src`?

**Comment by adamweeks** [comment]
This looks good. Did you verify the compiled code loads the fonts properly? I'm assuming it gets them from the momentum CDN.

We "solved" this issue on the widgets side with the following note in the readme:

>2. Add the following import statements

>  ```js
> import RecentsWidget from '@ciscospark/widget-recents';
>  // Sass import required for styling widgets
>  import '@ciscospark/widget-recents/src/momentum.scss';
>  ```

The good thing about doing it this way is the fonts and images all load locally from node_modules when an external user imports the components.

**Comment by akoushke** [comment]
> This looks good. Did you verify the compiled code loads the fonts properly? I'm assuming it gets them from the momentum CDN.
> 
> We "solved" this issue on the widgets side with the following note in the readme:
> 
> > 1. Add the following import statements
> 
> > ```js
> > import RecentsWidget from '@ciscospark/widget-recents';
> > // Sass import required for styling widgets
> > import '@ciscospark/widget-recents/src/momentum.scss';
> > ```
> 
> The good thing about doing it this way is the fonts and images all load locally from node_modules when an external user imports the components.

fonts are being loaded fine. no warnings or errors here
Screen Shot 2019-09-11 at 10 02 03 AM

File: README.md [hunk 3]
```diff
@@ -29,84 +29,78 @@
...
 
-2. Import `@momentum-ui/core/scss/momentum-ui-components` into your main entry Sass file AFTER the variables.
+```js
+import '@webex/components/dist/webexComponents.css';
+```
 
-    ```scss
-    @import "~@momentum-ui/core/scss/momentum-ui-components";
-    ```
+### 2. Html
```
Comment (lalli-flores) [inline README.md]: HTML

File: README.md [hunk 4]
```diff
@@ -29,84 +29,78 @@
...
-  function App() {
-    return (
-      <div className='App'>
-        <WebexAvatar title='Webex Avatar' />
-      </div>
-    );
-  }
+function App() {
+  return (
+    <div className="App">
```
Comment (lalli-flores) [inline README.md]: Why did you change to `"`? There's still `'` used in the imports

Comment (akoushke) [reply README.md]: prettify did that.
